contributing/README.md

```diff
@@ -34,6 +34,15 @@ Start the title with `[ComponentName]` to identify which component a change affe
 [FooBar] Removes the deprecated fooWithBar:(Bar*)bar method.
 ```
 
+### Pull request continuous integration for new contributors
+
+Continuous integration will only be initiated automatically for the core team and
+recognized collaborators.
+
+All other pull requests must be labeled with `kokoro:force-run` by a member of the repo
+in order for continuous integration to be initiated. This label must be added again each
+time the pull request has new commits pushed to it.
```

Comment:
This seems like additional work for the team.

Can we not have CI run on all PRs?
Is the concern that external contributors won't be able to open detailed results?

Reply:
Unfortunately this is not something we'll be able to work around - see b/115506452 for additional details.

Reply:
Got it, this is a requirement.